internal/service/s3/bucket_new.go

```diff
@@ -31,12 +31,12 @@ import (
 	"github.com/hashicorp/terraform-provider-aws/internal/verify"
 )
 
-func ResourceBucket() *schema.Resource {
+func ResourceBucketNew() *schema.Resource {
```

Comment:
Probably okay - but curious - why did we move and change names in these files?  Couldn't we have left these files unmodifed (to reduce diff to upstream), and just pulled the new files in in the seperate folder as done here?  Why did anything in these files need to change?